test/uk/gov/hmrc/cdsimportsddsfrontend/controllers/DeclarationControllerSpec.scala

```diff
@@ -215,15 +298,39 @@ class DeclarationControllerSpec extends CdsImportsSpec
         "documentationType.previousDocGoodsItemId" -> Seq("1"),
         "documentationType.additionalInfoCode" -> Seq("00500"),
         "documentationType.additionalInfoDescription" -> Seq("IMPORTER"),
-        "documentationType.additionalDocCategoryCode" -> Seq("N"),
-        "documentationType.additionalDocTypeCode" -> Seq("935"),
-        "documentationType.additionalDocId" -> Seq("12345/30.09.2019"),
-        "documentationType.additionalDocLPCO" -> Seq("AC"),
-        "documentationType.additionalDocName" -> Seq("DocumentName"),
+        "documentationType.additionalDocument[0].categoryCode" -> Seq("N"),
+        "documentationType.additionalDocument[0].typeCode" -> Seq("935"),
+        "documentationType.additionalDocument[0].id" -> Seq("12345/30.09.2019"),
+        "documentationType.additionalDocument[0].lpco" -> Seq("AC"),
+        "documentationType.additionalDocument[0].name" -> Seq("DocumentName"),
```

Comment:
If section 2 fields are now optional we should also test that the fields can all be submitted with no data in them

Reply:
Agree.
No tests existed for section 2 at all.